docs/sources/panels/visualizations/state-timeline.md

```diff
@@ -41,7 +41,7 @@ Controls the opacity of state regions.
 
 ## Value mappings
 
-To assign colors to boolean or string values, use the [Value mappings](< {{ refref "../value-mappings.md"}} >).
+To assign colors to boolean or string values, use [Value mappings]({{< relref "./value-mappings.md" >}}).
```

Comment:
Wait, what? This change shouldn't work. Value mappings is up a level from the visualizations files, so it should require two dots, not one.

Reply:
The older relref [Value mappings](< {{ refref "../value-mappings.md"}} >) did not work. Any suggestion?

Reply:
I see it working in my local build. Maybe an issue with the quotes ...